indexing-solr/src/test/resources/datastream/inventoryMods.xml

```diff
@@ -6,6 +6,13 @@
   <mods:genre>Dissertation</mods:genre>
   <mods:subject>
     <mods:topic>Test resource</mods:topic>
+    <mods:name>
```

Comment:
Checked with anna about this, it sounds like if there were multiple subjects on a record they would be under separate `mods:subject` tags, but if this situation ever did occur the behavior in this PR of creating multiple solr subject entries is what we would want. So maybe keep this block as is, but also add a second `mods:subject/mods:topic` to verify that it will work with multiple top level subject tags?